File: materi/pertemuan_4/java/Input.java
import java.util.Scanner;

public class Input {
    public static void main(String[] args) {
        Scanner newScanner = new Scanner(System.in);
        System.out.println("Masukan nama: ");

        String nama = newScanner.nextLine();
        System.out.println("nama: " + nama);
    }
}

File: materi/pertemuan_4/kotlin/input.kt
package pertemuan_4

import java.util.*

fun main () {

    //membuat objek scanner
    val newScanner = Scanner (System.`in`)

    //mengambil input
    print("Masukan Nama : ")
    val nama = newScanner.nextLine()

    print("Masukan nim : ")
    val nim = newScanner.nextLine()

    //menampilkan isi nama dan nim
    println("NIM Mahasiswa dengan nama $nama adalah $nim")

}